src/bsoncxx/json.hpp

```diff
@@ -36,28 +36,6 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 BSONCXX_API std::string BSONCXX_CALL to_json(document::view view);
 
 ///
-/// Converts an element (key-value pair) to a JSON key-value pair.
-/// This is not actually a valid JSON document, but it can be useful for debugging.
-///
-/// @param element
-///   A BSON element.
-///
-/// @returns A JSON key-value pair.
-///
-BSONCXX_API std::string BSONCXX_CALL to_json(document::element element);
```

Comment:
What about creating a builder, appending the single element, then dumping that to JSON?  It changes the semantics (e.g. now a valid JSON document) but still has debugging utility.